parsl/executors/high_throughput/executor.py

```diff
@@ -627,46 +627,40 @@ def scale_in(self, blocks=None, block_ids=[], force=True, max_idletime=None):
              A time to indicate how long a block can be idle.
              Used along with force = False to kill blocks that have been idle for that long.
 
-        block_ids : list
-             List of specific block ids to terminate. Optional
-
         Returns
         -------
         List of job_ids marked for termination
         """
-        logger.debug(f"Scale in called, blocks={blocks}, block_ids={block_ids}")
-        if block_ids:
-            block_ids_to_kill = block_ids
+        logger.debug(f"Scale in called, blocks={blocks}")
+        managers = self.connected_managers()
```

Comment:
"While you're here," and changing these lines anyway, do you want to continue on the path of not pre-formatting these log messages?  That is:
```python
logger.debug("Scale in called, blocks=%s", blocks)
```
Not a blocker, but a whittling away of that other task ... "while here anyway."

Reply:
would be an interesting linting rule to write on a lazy sunday afternoon...